### PR TITLE
Adjusted mudcrab to be more energetic and brave

### PR DESCRIPTION
**Pull request by Noggog**
Did the NPC AI pass, as requested. 

Main difference was I made the mudcrab more energetic and brave than it originally was, since it's so big, it's probably not scared of anything.

### CODE REVIEW

**Comment by Noggog** [comment]
Did you decrease the energy level?

**Comment by Noggog** [comment]
Cool thanks for the change!

File: Npcs/SpriggitMudcrab.yaml [hunk 1]
```diff
@@ -23,8 +23,8 @@ Configuration:
 Template: 02183E:Skyrim.esm
 Race: 109C7C:Skyrim.esm
 AIData:
-  Confidence: Average
-  EnergyLevel: 50
+  Confidence: Brave
+  EnergyLevel: 55
```
Comment (dragonjet) [inline Npcs/SpriggitMudcrab.yaml:27]: maybe more, like 60?

Comment (Noggog) [reply Npcs/SpriggitMudcrab.yaml:27]: lol

Comment (halgari) [reply Npcs/SpriggitMudcrab.yaml:27]: why are you so anti-mudcrabs, make it > 9000

**Pull request merged**